recipes/voyager/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set version = "0.1.0" %}
+{% set base = "https://bitbucket.org/sverre-phd-work/voyager/downloads" %}
+{% set sha256 = "57a87fb9878a25a3848c1bde6f6f873fd154ff6f1b229b09afccf602d3ea82a9" %}
+
+package:
+  name: voyager
+  version: '{{version}}'
+
+source:
+  url: {{base}}/voyager-linux64-v{{version}}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('voyager', max_pin="x.x") }}
+
+requirements:
+  build:
+    #
+  host:
+    - openjdk >=11
+  run:
+    - openjdk >=11
+
+about:
+  home: "https://bitbucket.org/sverre-phd-work/voyager/"
+  license: LGPLv3
```

```suggestion
  license: LGPL-3.0-only
```
Please use SPDX identifier https://spdx.org/licenses/